apisix/http/router/radixtree_host_uri.lua

```diff
@@ -130,16 +131,17 @@ function _M.match(api_ctx)
     match_opts.remote_addr = api_ctx.var.remote_addr
     match_opts.vars = api_ctx.var
     match_opts.host = api_ctx.var.host
+    match_opts.matched = {}
```

Comment:
Can we reuse this Lua table?

Reply:
you mean using `tablepool`? I get, the `api_ctx` also use `tablepool`, let me try.

Reply:
yes, waiting for your result

Reply:
suspend merged, let me check the Grafana Dashboard

Reply:
I find that this change will not be reflected in Grafana Dashboard, may be because of `apisix-grafana-dashboard`  hasn't been updated, but I don't know how to modify it yet.